website/README.md

```diff
@@ -118,6 +118,42 @@ The code for the client is located under `src/client/`. The webpack build create
 
 Scripts related to deployment, like the Dockerfile, are under `deploy`. Note: even though the scripts are under `deploy`, you must run them from the **root of the repository**; they are sensitive to `cwd`.
 
+## Query Parameters
+
+You can pass certain parameters in the Snack URL to customize the behavior of the Snack.
+
+Here is a summary of all the parameter options that you can use:
+
+| parameter | default | description | type |
+| -------- | ------- | ----------- | ------ |
+| **dependencies** | | Dependencies to be used by your Snack beyond common dependencies and the ones added in the Snack's `package.json`. [Read more about Snack dependencies](_`string[https://github.com/expo/snack/blob/main/docs/snack-sdk.md#using-dependencies]`_). | [SnackDependency](https://github.com/expo/snack/blob/main/packages/snack-content/src/types.ts#L64) |
+| **description** | | The description of your Snack. | `string` |
+| **files** | | Allows you to inline files to load through a query param, useful for doc maintainers who don't want to maintain twice the code (both markdown and snack, itself). [Read more about files](https://github.com/expo/snack/blob/main/docs/snack-sdk.md#files-dependencies-and-the-state). | [`SnackFiles`](https://github.com/expo/snack/blob/main/packages/snack-content/src/types.ts#L40) |
+| **name** | | The name of your Snack. When creating a new Snack, a random name is assigned unless one is provided via this parameter.  | `string` |
+| **platform** | `web` | The platform on which your Snack should be run in the Device Preview window. | `android`, `ios`, `mydevice`, `web` |
+| **preview** | `true` | Toggle to show the Device Preview. | `boolean` |
+| **sdkVersion** | `44.0.0` | The Expo SDK version that your Snack should use. | `45.0.0`, `44.0.0`, `43.0.0` |
+| **snackId** | | The unique ID of your Snack. | `string` |
+| **supportedPlatforms** | All platforms | Specify which platforms your Snack supports | `android`, `ios`, `mydevice`, `web` |
+| **theme** | `light` | The visual theme of your Snack. | `light`, `dark` |
+
+## Examples of Snack URLs
+
+All of these examples should be prefixed with `https://snack.expo.dev/`.
+
+| parameter | example |
+| --------- | ------- |
+| `name` | `?name=Gordon%20Freeman` |
+| `description` | `?description=Research%20assistant%20at%20the%20Black%20Mesa%20Research%20Facility.` |
+| `snackId` |  |
+| `platform` | `?platform=mydevice` |
+| `preview` | `?preview=false` |
+| `sdkVerion` | `?sdkVersion=45.0.0` |
+| `supportedPlatforms` | `?supportedPlatforms=android,ios` |
+| `theme` | `?theme=dark` |
+| `dependencies` | `?dependencies=%40expo%2Fvector-icons%40*%2C%40react-native-community%2Fmasked-view` |
+| `files` | `?files=` |
```

Comment:
For these examples, I think we can make it a bit shorter. What do you think of listing a couple, e.g. 3, URLs and explaining each param set? E.g. something like:

---

### Example URLs

- [?name=Gordon%20Freeman&description=Where%20is%20Halflife%203](https://snack.expo.dev/?name=Gordon%20Freeman&description=Where%20is%20Halflife%203)
  `name` is set to **Gordon Freeman**
  `description` is set to **Where is Halflife 3**

- [?preview=false&theme=dark](https://snack.expo.dev/?preview=false&theme=dark)
  `preview` is **false**, hiding the right "preview" panel
  `theme` is **dark**

---

I'm also ok with a table, but let's try and make that a bit cleaner.
- Try and use **bold** for query param names, I think it reads a bit nicer plus they should be double-click selectable as they are 
URL safe (no spaces). You already did this nicely on the query param table
- Make the example links clickable (just like I did in the example above).

Reply:
I'll clean up the table for now, I think it's important to show examples for `files` and `dependencies`, in particular. I'd be down to do 3 examples if they included those two. Lmk what you think about adding those (seems like linking `files` might not work unless we set up a hosted file).

Reply:
Lets leave it then :)